servers/audio/audio_server.h

```diff
@@ -232,6 +232,9 @@ class AudioServer : public Object {
 	bool debug_mute = false;
 #endif // DEBUG_ENABLED
 
+	bool input_device_active = false;
```

Comment:
Do we need a separate boolean for this? Can't we ask the audio driver instead if it's active? I'm always a bit wary of dual states that aren't enforceable.